### PR TITLE
Fix: setting view 

### LINKED ISSUE #45: split the setting view in different view
## Problem

Our Setting View is not showing a very nice UI because is developed in the wrong way!

## Backgbround

This task required reworking the Setting View by splitting the sub-view in different Widget as suggested in this issue https://github.com/dart-lightning/lndart.clnapp/pull/35#discussion_r944285954

## Goal

Make the setting view code clean to read and also easy to work on it

## Object of this tasks

- Show that you are able to understand how Flutter works, and not just make some code that works
- Show that you are able to work with git, and follow the contributor guideline https://github.com/dart-lightning/lndart.clnapp/blob/main/docs/dev/MAINTAINERS.md
- Show that you can propose and improve the application with your own idea from a general one

### PR DESCRIPTION
**Pull request by Harshit933**
Created seperate files for different views in the `settings_view.dart` namely

`lib/views/setting/grpc_settings.dart`
`lib/views/setting/lnlambda.dart`
`lib/views/setting/unix_settings.dart`

Also moved the directory picker function into another file under `lib/utils/directory_picker.dart`
 
Fixes #45

### CODE REVIEW

**Comment by vincenzopalazzo** [comment]
can you call all the file <current name>_view.dart?

File: lib/views/setting/lnlambda_setting.dart [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import 'package:flutter/material.dart';
+
+import '../../model/user_setting.dart';
+import '../../utils/app_provider.dart';
+
+class LnlambdaSetting extends StatefulWidget {
```
Comment (vincenzopalazzo) [inline lib/views/setting/lnlambda_setting.dart:6]:
```suggestion
class LNlambdaSettingView extends StatefulWidget {
```

File: lib/views/setting/unix_setting.dart [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+
+import '../../model/user_setting.dart';
+import '../../utils/app_provider.dart';
+import '../../utils/directory_picker.dart';
+
+class UnixSetting extends StatefulWidget {
```
Comment (vincenzopalazzo) [inline lib/views/setting/unix_setting.dart:7]:
```suggestion
class UnixSettingView extends StatefulWidget {
```

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]

**Comment by Harshit933** [comment]
> can you call all the file _view.dart?

On it

**Comment by vincenzopalazzo** [comment]
I think this is good to go,  but now you need to squash the commit in a single one and follow the maintainer rule https://github.com/dart-lightning/lndart.clnapp/blob/main/docs/dev/MAINTAINERS.md

Sorry about my commit rules :)

**Comment by Harshit933** [comment]
Done! I GOT IT NOW! THANK YOU SO MUCH for letting me know such features exists in github.

**Review by vincenzopalazzo — APPROVED** [review]
Much more readble, thanks!

**Pull request merged**